examples/bert/bert_inc_static_ptq_cpu.json

```diff
@@ -12,6 +12,15 @@
                 "max_samples": 100
             },
             "dataloader_config": { "batch_size": 1 }
+        },
+        {
+            "name": "inc_quat_data_config",
+            "type": "HuggingfaceContainer",
+            "user_script": "user_script.py",
+            "load_dataset_config": { "type": "bert_inc_glue_calibration_dataset" },
+            "pre_process_data_config": { "type": "skip_pre_process" },
+            "post_process_data_config": { "type": "skip_post_process" },
+            "dataloader_config": { "type": "bert_inc_glue_calibration_dataloader" }
```

Comment:
Since the logics to generate dataset are defined by users, for default behavior, how about just keep `dataloader_config`? That says, by default:
```json
"load_dataset_config": { "type": "skip_dataset" },
"pre_process_data_config": { "type": "skip_pre_process" },
"post_process_data_config": { "type": "skip_post_process" },
"dataloader_config": { "type": "bert_inc_glue_calibration_dataloader" }
```

Then user just need to register dataloader component.

Reply:
I think this behavior can be achieved by just defining the data config as:
```json
{
    "name": "inc_quant_data_config", 
    "user_script": "user_script.py",
    "load_dataset_config": { "type": "bert_inc_glue_calibration_dataset" },
    "dataloader_config": { "type": "bert_inc_glue_calibration_dataloader" }
}
```

When type is not defined, it uses the default data container. this container's default load, preprocess and postprocess components are the same as skip.
setting type as `HuggingfaceContainer` when everything is overriden is misleading.

Reply:
this also applies to the examples where only the data loader is registered and everything else is skipped.

Reply:
Fixed.